baikal-vdu: Selecting to build as modules by default for baikal arch

From: Vitaly Miniakhmetov <vitalii.miniakhmetov@t-platforms.ru>

diff --git a/arch/arm64/configs/tp_baikalm_defconfig b/arch/arm64/configs/tp_baikalm_defconfig
index 4b732e1d8bee..00f630c6b984 100644
--- a/arch/arm64/configs/tp_baikalm_defconfig
+++ b/arch/arm64/configs/tp_baikalm_defconfig
@@ -311,8 +311,7 @@ CONFIG_DRM_NOUVEAU=m
 CONFIG_DRM_VIRTIO_GPU=m
 CONFIG_DRM_PANEL_LVDS=m
 CONFIG_DRM_STDP4028=y
-CONFIG_DRM_DW_HDMI_AHB_AUDIO=y
-CONFIG_DRM_BAIKAL_HDMI_AHB_AUDIO=y
+CONFIG_DRM_DW_HDMI_AHB_AUDIO=m
 CONFIG_DRM_PANFROST=m
 CONFIG_FB_MODE_HELPERS=y
 CONFIG_FB_EFI=y
diff --git a/drivers/gpu/drm/baikal/Kconfig b/drivers/gpu/drm/baikal/Kconfig
index bf4a44283c99..2bac5e3a181c 100644
--- a/drivers/gpu/drm/baikal/Kconfig
+++ b/drivers/gpu/drm/baikal/Kconfig
@@ -3,6 +3,7 @@ config DRM_BAIKAL_VDU
 	depends on DRM
 	depends on ARM || ARM64 || COMPILE_TEST
 	depends on COMMON_CLK
+	default m if ARCH_BAIKAL
 	select DRM_KMS_HELPER
 	select DRM_KMS_CMA_HELPER
 	select DRM_GEM_CMA_HELPER
diff --git a/drivers/gpu/drm/bridge/synopsys/Kconfig b/drivers/gpu/drm/bridge/synopsys/Kconfig
index 56346a0e9dc6..2f914d45755e 100644
--- a/drivers/gpu/drm/bridge/synopsys/Kconfig
+++ b/drivers/gpu/drm/bridge/synopsys/Kconfig
@@ -42,6 +42,7 @@ config DRM_DW_MIPI_DSI
 
 config DRM_BAIKAL_HDMI
 	tristate "Baikal-M HDMI transmitter"
+	default m if ARCH_BAIKAL
 	select DRM_DW_HDMI
 	help
 	  Choose this if you want to use HDMI on Baikal-M.
@@ -49,6 +50,7 @@ config DRM_BAIKAL_HDMI
 config DRM_BAIKAL_HDMI_AHB_AUDIO
 	tristate "Baikal-M HDMI Audio interface"
 	depends on DRM_BAIKAL_HDMI && SND
+	default m if ARCH_BAIKAL
 	select SND_PCM
 	select SND_PCM_ELD
 	select SND_PCM_IEC958
